e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -559,51 +559,50 @@ public async Task ProvisioningDeviceClient_ValidRegistrationId_MqttWsWithProxy_S
+                transport);
+            using var cts = new CancellationTokenSource(PassingTimeoutMiliseconds);
+
+            // Test registering with valid additional data payload
+            DeviceRegistrationResult result = await provClient
+                .RegisterAsync(new ProvisioningRegistrationAdditionalData { JsonData = PayloadJsonData }, cts.Token)
+                .ConfigureAwait(false);
+            ValidateDeviceRegistrationResult(true, result);
+            Assert.AreEqual(expectedDestinationHub, result.AssignedHub);
+
+            // Test registering without additional data
+            result = await provClient.RegisterAsync(cts.Token).ConfigureAwait(false);
+            ValidateDeviceRegistrationResult(false, result);
+
+            if (attestationType != AttestationMechanismType.X509) //x509 enrollments are hardcoded, should never be deleted
             {
-                //Check basic provisioning
-                if (ImplementsWebProxy(transportProtocol) && setCustomProxy)
+                try
                 {
-                    transport.Proxy = (proxyServerAddress != null) ? new WebProxy(s_proxyServerAddress) : null;
+                    await DeleteCreatedEnrollmentAsync(enrollmentType, provisioningServiceClient, security, groupId).ConfigureAwait(false);
                 }
-
-                var provClient = ProvisioningDeviceClient.Create(
-                    s_globalDeviceEndpoint,
-                    Configuration.Provisioning.IdScope,
-                    security,
-                    transport);
-                using var cts = new CancellationTokenSource(PassingTimeoutMiliseconds);
-
-                //Test registering with valid additional data payload
-                DeviceRegistrationResult result = await provClient
-                    .RegisterAsync(new ProvisioningRegistrationAdditionalData { JsonData = PayloadJsonData }, cts.Token)
-                    .ConfigureAwait(false);
-                ValidateDeviceRegistrationResult(true, result);
-                Assert.AreEqual(expectedDestinationHub, result.AssignedHub);
-
-                //Test registering without additional data
-                result = await provClient.RegisterAsync(cts.Token).ConfigureAwait(false);
-                ValidateDeviceRegistrationResult(false, result);
-
-                if (attestationType != AttestationMechanismType.X509) //x509 enrollments are hardcoded, should never be deleted
+                catch (Exception ex)
                 {
-                    try
-                    {
-                        await DeleteCreatedEnrollmentAsync(enrollmentType, provisioningServiceClient, security, groupId).ConfigureAwait(false);
-                    }
-                    catch (Exception ex)
-                    {
-                        Console.WriteLine($"Cleanup of enrollment failed due to {ex}");
-                    }
+                    Console.WriteLine($"Cleanup of enrollment failed due to {ex}");
```

Comment:
This was also already there but curious why not Logger.Trace. We should probably update.

Reply:
I thought Logger.Trace was only for SDK code, not tests. In a test, Console.WriteLine goes to test output.

Reply:
Oh, is that the test logger?